src/Utility/StringUtil.ts

```diff
@@ -132,4 +133,20 @@ export class StringUtil {
             && s2Type !== "undefined"
             && s1.toLowerCase() === s2.toLowerCase();
     }
+
+    public static escapeString(builder: StringBuilder, value: string) {
```

Comment:
It doesn't really say what does it escape. Is it for escaping strings in RQL?

Reply:
copied 1:1 from .net and java.